.ci_support/osx_mpinompi.yaml

```diff
@@ -1,3 +1,5 @@
+CONDA_BUILD_SYSROOT:
+- /opt/MacOSX10.10.sdk
```

Comment:
Where did this come from?

Reply:
No idea. But somehow OS X is passing, the problem is with Linux :smile: